wolox-training/rof-angular
# Implemented book suggestions

## Summary

Implemented book suggestions

## Test Plan

* Log into your account
* Go to suggestions trough the header's button
* Add a new suggestion
* An error message should show if the suggestion can't be made
* Form should clean up if suggestion was successfully sent (Will implement growl later)

## Trello Card 

https://trello.com/c/5yQpCLBt

## Screenshot

![suggestion](https://cloud.githubusercontent.com/assets/2911971/10741393/550adc78-7c06-11e5-9a83-0ac295179597.png)
